App getting crashed, if we upload a video of more than 20 minutes.

If we upload a video of more than 20 minutes, app is getting crashed, we have tested the video of duration 30 sec, 01 minute, 05 minutes, 10 minutes and 15 minutes, it's compressing fine, the video more than of 20 minutes is not compressing and giving us this log (I'm attaching the log of the RN-Debugger that we get, if we try to compress the 20min video). Kindly have a look into this.

-----------Debugger Logs---------------------------------
Failed to convert base64: Cannot Open
VideoCompressionScreen.js:91 error TypeError: Cannot read property 'path' of undefined
at _callee3$ (VideoCompressionScreen.js:80)
at tryCatch (runtime.js:63)
at Generator.invoke [as _invoke] (runtime.js:294)
at Generator.next (runtime.js:119)
at tryCatch (runtime.js:63)
at invoke (runtime.js:155)
at runtime.js:165
at tryCallOne (core.js:37)
at core.js:123
at JSTimers.js:248

--- a/example/src/Screens/Video/index.tsx
+++ b/example/src/Screens/Video/index.tsx
@@ -114,7 +114,6 @@ export default function App() {
       {
         compressionMethod: 'auto',
         minimumFileSizeForCompress: 5,
-        maxSize: 1000,
       },
       (progress) => {
         if (backgroundMode) {

--- a/ios/Video/VideoCompressor.swift
+++ b/ios/Video/VideoCompressor.swift
@@ -247,7 +247,7 @@ func makeValidUri(filePath: String) -> String {
         let maxBitrate:Int = 1669000
         let minValue:Float=min(Float(originalHeight)/Float(height),Float(originalWidth)/Float(width))
         var remeasuredBitrate:Int = Int(Float(originalBitrate) / minValue)
-        remeasuredBitrate = remeasuredBitrate*Int(compressFactor)
+        remeasuredBitrate = Int(Float(remeasuredBitrate)*compressFactor)
         let minBitrate:Int = self.getVideoBitrateWithFactor(f: minCompressFactor) / (1280 * 720 / (width * height))
         if (originalBitrate < minBitrate) {
           return remeasuredBitrate;

